src/svc-hooks/create.ts

```diff
@@ -41,35 +41,40 @@ export async function handler(argv: any) {
         consumerId: "webHooks",
         consumerInputs: {
           url: hook.url,
+          acceptUntrustedCerts: hook.eventSpecificArgs.acceptUntrustedCerts,
         },
         eventType: hook.eventType,
         publisherId: hook.eventSpecificArgs.publisherId,
         publisherInputs: {},
-        resourceVersion: "1.0",
+        "resourceVersion": "5.1-preview.1",
         scope: 1,
       };
 
       // Check if eventType is valid, populated publisherInputs
       switch (hook.eventType) {
         case "build.complete":
+          hookDataFormatted.consumerInputs.acceptUntrustedCerts = hook.eventSpecificArgs.acceptUntrustedCerts;
           hookDataFormatted.publisherInputs = {
-            definitionName: hook.eventSpecificArgs.definitionName,
             buildStatus: hook.eventSpecificArgs.buildStatus,
+            definitionName: hook.eventSpecificArgs.pipelineName,
+            projectId: await searchProjId(hook.org, hook.project),
           };
           break;
         case "ms.vss-pipelines.stage-state-changed-event":
           hookDataFormatted.publisherInputs = {
-            pipelineId: hook.eventSpecificArgs.pipelineId,
+            pipelineId: hook.eventSpecificArgs.buildDefinitionId, // TODO: Figure out which input is best and how to get pipelineId from it.
```

Comment:
I'm confused what this TODO is supposed to mean...

Reply:
Whoops, this is a leftover comment for me when I was figuring out the buildDefinitionId input, I'll remove this. Not necessary.